BossMod/Data/PartyState.cs

```diff
@@ -45,9 +45,9 @@ void assign(ulong instanceID, Actor? actor)
     }
 
     // select non-null and optionally alive raid members
-    public IEnumerable<Actor> WithoutSlot(bool includeDead = false)
+    public IEnumerable<Actor> WithoutSlot(bool includeDead = false, bool partyOnly = true)
```

Comment:
This is a big breaking change, why do you need it?

Reply:
oh right, good catch. i'll change the default value to be false